client/app/telecom/telephony/billingAccount/administration/addGroup/telecom-telephony-billing-account-administration-add-group.html

```diff
@@ -11,58 +11,46 @@ <h1 data-translate="telephony_add_group_title"></h1>
         <oui-spinner></oui-spinner>
     </div>
 
-    <div class="telecom-section-content"
+    <div class="telecom-section-content col-lg-11"
          data-ng-if="!AddGroupCtrl.loading">
-
         <toast-message></toast-message>
-
-        <div class="row">
-            <div class="col-md-6 mb-5">
-
-                <div data-ng-if="!AddGroupCtrl.order">
-                    <p class="alert alert-info"
-                       role="alert"
-                       data-translate="telephony_add_group_infos">
-                    </p>
-                    <p data-translate="telephony_add_group_contracts"></p>
-                    <div data-contracts="AddGroupCtrl.contracts"
-                         data-full-text="true"
-                         data-contracts-validated="AddGroupCtrl.contractsAccepted">
-                    </div>
-                    <button type="button"
-                            class="btn btn-primary"
-                            data-ng-disabled="!AddGroupCtrl.contractsAccepted || AddGroupCtrl.ordering"
-                            data-ng-click="AddGroupCtrl.orderGroup()">
-                        <oui-spinner class="mr-2"
-                                     data-ng-if="AddGroupCtrl.ordering"
-                                     data-size="s">
-                        </oui-spinner>
-                        <span data-translate="telephony_add_group_order"></span>
-                        <span>({{ AddGroupCtrl.getDisplayedPrice() }})</span>
-                    </button>
-                </div>
-
-                <div data-ng-if="AddGroupCtrl.order">
-                    <p>
-                        <i class="ovh-font ovh-font-success text-success mb-0 mr-2"></i>
-                        <span data-translate="telephony_add_group_success"></span>
-                    </p>
-                    <p>
-                        <span data-translate="telephony_add_group_success_bc"></span>
-                        <a target="_blank"
-                           rel="noopener"
-                           data-ng-href="{{ AddGroupCtrl.order.url }}">
-                            <span data-translate="telephony_add_group_success_bc2"></span>
-                            <i class="fa fa-external-link"></i>
-                        </a>
-                        .
-                    </p>
-                </div>
+        <div data-ng-if="!AddGroupCtrl.order">
+            <p class="alert alert-info"
+               role="alert"
+               data-translate="telephony_add_group_infos">
+            </p>
+            <p data-translate="telephony_add_group_contracts"></p>
+            <div data-contracts="AddGroupCtrl.contracts"
+                 data-full-text="true"
+                 data-contracts-validated="AddGroupCtrl.contractsAccepted">
             </div>
+                <button class="oui-button oui-button_primary oui-button_full-width col-md-7 col-lg-5"                       
```

Comment:
Could you please fix indentation? :)

And you can refer to: https://github.com/twbs/bootlint/wiki/E014